testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -52,16 +53,21 @@ private void Awake()
 
     public void Start()
     {
+        BaseMultiprocessTests.MultiProcessLog("Start");
         bool isClient = Environment.GetCommandLineArgs().Any(value => value == MultiprocessOrchestration.IsWorkerArg);
         if (isClient)
         {
+            NetworkManager.NetworkConfig.EnableNetworkLogs = true;
```

Comment:
do we want this for all multiprocess tests? logs will slow down perf tests for example

Reply:
okay, I'll remove it, I was just trying to figure out why sometimes connections never established. Perhaps we can put it back when we actually have a problem. Where do these logs go anyhow?

Reply:
addressed in https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/pull/1195/commits/4d26c46c0d7b955b296b3162b28c995fdd2c45b3